### PR TITLE
CLOUD-513 Add license tests

### PR DESCRIPTION
**Pull request by hors**

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| scaling | passed |
| big-data | passed |
| self-healing | passed |
| init-deploy | passed |
| recreate | failed |
| scaling-proxysql | passed |
| storage | failed |
| self-healing-advanced | failed |
| upgrade | passed |
| operator-self-healing | passed |
| demand-backup | failed |
| limits | passed |
| upgrade-consistency | passed |
| monitoring | passed |
| one-pod | passed |
| auto-tuning | passed |
| monitoring-2-0 | passed |
| scheduled-backup | passed |
| affinity | passed |
| proxysql-sidecar-res-limits | passed |

File: Jenkinsfile [hunk 1]
```diff
@@ -137,6 +137,13 @@ pipeline {
                         | sudo tar -C /usr/local/bin --strip-components 1 -zvxpf -
                     curl -s -L https://github.com/openshift/origin/releases/download/v3.11.0/openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit.tar.gz \
                         | sudo tar -C /usr/local/bin --strip-components 1 --wildcards -zxvpf - '*/oc'
+                    
+                    curl -s -L https://github.com/mitchellh/golicense/releases/latest/download/golicense_0.2.0_linux_x86_64.tar.gz \
+                        | sudo tar -C /usr/local/bin --wildcards -zxvpf -
+                    curl -s -L https://github.com/src-d/go-license-detector/releases/latest/download/license-detector.linux_amd64.gz \
+                        | gunzip > license-detector
```
Comment (delgod) [inline Jenkinsfile]:
```suggestion
                        | gunzip > /usr/local/bin/license-detector
```

File: Jenkinsfile [hunk 2]
```diff
@@ -172,7 +179,43 @@ pipeline {
...
+                expression {
+                    !skipBranchBulds
+                }
+            }
+            steps {
+                sh '''
+                    mkdir -p $WORKSPACE/src/github.com/percona
+                    ln -s $WORKSPACE $WORKSPACE/src/github.com/percona/percona-xtradb-cluster-operator
+                    sg docker -c "
+                        docker run --rm -v $WORKSPACE/src/github.com/percona/percona-xtradb-cluster-operator:/go/src/github.com/percona/percona-xtradb-cluster-operator -w /go/src/github.com/percona/percona-xtradb-cluster-operator -e GO111MODULE=on golang:1.12 sh -c 'go mod init && go build ./... && rm -rf Gopkg.lock Gopkg.toml vendor && go build -v ./... && go build -o percona-xtradb-cluster-operator github.com/percona/percona-xtradb-cluster-operator/cmd/manager'
```
Comment (delgod) [inline Jenkinsfile]:
```suggestion
                        docker run \
                            --rm \
                            -v $WORKSPACE/src/github.com/percona/percona-xtradb-cluster-operator:/go/src/github.com/percona/percona-xtradb-cluster-operator \
                            -w /go/src/github.com/percona/percona-xtradb-cluster-operator \
                            -e GO111MODULE=on \
                            golang:1.12 sh -c ' \
                                go mod init \
                                    && go build ./... \
                                    && rm -rf Gopkg.lock Gopkg.toml vendor \
                                    && go build -v ./... \
                                    && go build -o percona-xtradb-cluster-operator github.com/percona/percona-xtradb-cluster-operator/cmd/manager
                            '
```

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |

File: Jenkinsfile (hunk 2 above)
Comment (hors) [reply Jenkinsfile]: done

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| self-healing | passed |
| scaling | passed |
| big-data | failed |
| recreate | passed |
| init-deploy | passed |
| storage | failed |
| scaling-proxysql | passed |
| self-healing-advanced | passed |
| demand-backup | passed |
| upgrade | passed |
| operator-self-healing | passed |
| limits | passed |
| monitoring | passed |
| upgrade-consistency | passed |
| one-pod | passed |
| monitoring-2-0 | passed |
| scheduled-backup | passed |
| auto-tuning | passed |
| affinity | passed |
| proxysql-sidecar-res-limits | passed |

**Pull request merged**